shell.nix

```diff
@@ -159,6 +159,7 @@ haskell.project.shellFor {
     ${utillinux}/bin/taskset -pc 0-1000 $$
   ''
   + ''
+    export GITHUB_SHA=$(git rev-parse HEAD)
```

Comment:
@Pacman99 is this the correct place for this environment variable? I also took a look at the haskell.nix, and wasnt sure which one is the preferred one!